superset-frontend/src/SqlLab/components/TemplateParamsEditor.jsx

```diff
@@ -94,54 +74,45 @@ export default class TemplateParamsEditor extends React.Component {
         </a>{' '}
         syntax.
       </p>
-    );
-  }
+      <StyledConfigEditor
+        keywords={[]}
+        mode={language}
+        minLines={25}
+        maxLines={50}
+        onChange={onChange}
```

Comment:
here I think we can add a throttle or debounce from the lodash library so that we don't introduce any race conditions with the redux actions.